assets/charts/control-plane/packet-ccm/Chart.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v2
```

Comment:
Can we create an issue for upstream to provide/maintain the Helm chart?

Reply:
Created https://github.com/packethost/packet-ccm/issues/92